modules/dyn.js

```diff
@@ -0,0 +1,242 @@
+// dyn.js
+
+import {
+  pullComments, pullCommCounter, pushComment, pushLike, pullLikes,
+} from './socialApi.js';
+
+export default class DynGrid {
+  header = document.getElementById('Header');
+
+  footer = document.getElementById('Footer');
+
+  dynamicGrid = document.getElementById('dynamicGrid');
+
+  genCard = (data, i, unqId, imgSrc, foodName) => {
+    const id = i + 1;
+    const cardContainer = document.createElement('div');
+    const cardPic = document.createElement('div');
+    const cardPicImg = document.createElement('img');
+    cardPicImg.src = `${imgSrc}`;
+    const cardMeta = document.createElement('div');
+    const cardInfo = document.createElement('div');
+    const cardTitle = document.createElement('div');
+    cardTitle.innerHTML = `${foodName}`;
+    cardPic.classList.add('card-pic');
+    cardMeta.classList.add('card-meta');
+    cardInfo.classList.add('card-info');
+    cardTitle.classList.add('card-title');
+    const cardSocials = document.createElement('div');
+    cardSocials.classList.add('card-socials');
+    const likeBtn = document.createElement('img');
+    likeBtn.classList.add('like-btn');
+    likeBtn.src = '../src/icons/like.png';
```

Comment:
- [ ] The like icons are not visible on the page, please make sure to fix this issue.

### see the image for reference 

![image](https://user-images.githubusercontent.com/83803414/163669208-9b1b41a7-dc6f-4729-b7d3-a2c3b2267b37.png)